tests/integration/websocket/socket.test.ts

```
@@ -0,0 +1,352 @@
+
+      const checkComplete = (promptId: string) => {
+        resolvedPrompts.add(promptId);
+        if (resolvedPrompts.size === 2) {
+          ws1.close();
+          ws2.close();
+          resolve();
+        }
+      };
+
+      // Both clients should listen for both resolved prompts (broadcast goes to all)
+      const setupMessageListener = (ws: WebSocket, clientNum: number) => {
```

ESLint

[error] 'clientNum' is defined but never used.

(@typescript-eslint/no-unused-vars)